packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -86,6 +86,9 @@ interface IComboBoxOptionWrapperProps extends IComboBoxOption {
   /** True if the option is currently selected */
   isSelected: boolean;
 
+  /** True if the option is currently checked (multi-select) */
```

Comment:
>checked (multi-select) [](start = 38, length = 22)

What about the single selected checked case?

Reply:
Single select doesn't have a checked case I thought. Isn't multiselect the only one that cares about checked?